Vijay/Employee Admission Form/final.html

```diff
@@ -47,21 +46,6 @@
         var obj = JSON.parse(save);
         
         
-        
-        function myFunction() {
-          var input, filter, table, tr, td, i, txtValue;
-          input = document.getElementById("myInput");
-          table = document.getElementById("tabes1");
-          tr = table.getElementsByTagName("tr");
-          for (i = 0; i < obj.length; i++) {
-            td = obj[i].getElementsByTagName("td");
-            if (td)
-              {
-                obj[i].style.display = "";
-              }
-            }       
-          }
-        
 
         function backbut (){
```

Comment:
Move the code to separate JS file and indent the code properly

Reply:
Completed the given Reviews bro.